package.json

```diff
@@ -12,10 +12,11 @@
 		"node": ">=6.1"
 	},
 	"scripts": {
-		"test": "npm run lint && npm run compile-typescript && npm run test-unit",
+		"benchmarks": "node ./benchmarks/index.js",
+		"test": "npm run lint && npm run compile-typescript && npm run test-unit && npm run benchmarks",
 		"lint": "eslint .",
 		"test-unit": "jest",
-		"compile-typescript": "tsc index.d.ts --noImplicitAny",
+		"compile-typescript": "tsc index.d.ts --noImplicitAny --skipLibCheck",
```

Comment:
?

Reply:
It was complaining about Set

```
> tsc index.d.ts --noImplicitAny
node_modules/@types/node/index.d.ts:813:38 - error TS2304: Cannot find name 'Set'.
813         allowedNodeEnvironmentFlags: Set<string>;
                                         ~~~
```

https://travis-ci.org/siimon/prom-client/jobs/431769692

This appeared to fix it, but I honestly am not up to speed on typescript to know the incantations to get it to work properly. I am assuming that some dep that is now pulled in pulls in a new @types definition that `tsc` is choking on?

Reply:
Try to add `--target es6` instead

Reply:
Awesome. Thanks for the guidance.